### PR TITLE
adds nav for plugin options, adds meta, fixes typos, other fixes

### PR DESCRIPTION
**Pull request by r12s**

### CODE REVIEW

**Comment by r12s** [comment]
@kilianc or @androb i'll leave this PR to you please as there are updates to the plugins nav plus a new rails integration added

**Comment by kilianc** [comment]
@r12s looks good overall :+1:

**Comment by kilianc** [comment]
let's let this branch deploy on staging k ?

File: plugins/codesample.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Codesample Plugin
-title_nav: Code Sample
+title_nav: Codesample
```
Comment (androb) [inline plugins/codesample.md]: I'd prefer that we leave this as two words. There are lots of plugins that have had a similar treatment.

File: plugins/codesample.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ controls: toolbar button
 
 > New in 4.3!
 
-The Code Sample plugin (`codesample`) lets a user insert and embed syntax color highlighted code snippets into the editable area. It also adds a button to the toolbar which on click will open a dialog box to accept raw code input.
+The Codesample plugin (`codesample`) lets a user insert and embed syntax color highlighted code snippets into the editable area. It also adds a button to the toolbar which on click will open a dialog box to accept raw code input.
```
Comment (androb) [inline plugins/codesample.md]: Prefer that we leave this as two words.

**Comment by androb** [comment]
Aside from my comment re leaving Code Sample plugin as two words when referred to as a noun (vs. a setting) I am good with this to be pushed to staging!

**Pull request merged**